app/views/exercises/_read_only.html.erb

```diff
@@ -77,32 +73,30 @@
         </ul>
 
         <div class="tab-content">
-          <% if @discussion.has_submission? %>
-            <div role="tabpanel" class="tab-pane active" id="solution">
-              <div class="mu-tab-body">
-                <div role="tabpanel" class="tab-pane mu-input-panel fade show active" id="editor">
-                  <div class="mu-read-only-editor">
-                    <%= render_exercise_read_only_editor exercise, @discussion.solution %>
-                  </div>
+          <div role="tabpanel" class="tab-pane active" id="solution">
+            <div class="mu-tab-body">
+              <div role="tabpanel" class="tab-pane mu-input-panel fade show active" id="editor">
+                <div class="mu-read-only-editor">
+                  <%= render_exercise_read_only_editor exercise, @discussion.solution %>
                 </div>
               </div>
             </div>
+          </div>
 
-            <div role="tabpanel" class="tab-pane" id="results">
-              <div class="mu-tab-body">
-                <%= render layout: 'exercise_solutions/contextualization_results_container', locals: { contextualization: @discussion } do %>
-                  <div class="row">
-                    <div class="col-md-12 submission-results">
-                      <%= render partial: 'exercise_solutions/contextualization_results_body',
-                                 locals: { contextualization: @discussion, guide_finished_by_solution: false } %>
-                    </div>
+          <div role="tabpanel" class="tab-pane" id="results">
+            <div class="mu-tab-body">
+              <%= render layout: 'exercise_solutions/contextualization_results_container', locals: { contextualization: @discussion } do %>
+                <div class="row">
+                  <div class="col-md-12 submission-results">
+                    <%= render partial: 'exercise_solutions/contextualization_results_body',
+                               locals: { contextualization: @discussion, guide_finished_by_solution: false } %>
                   </div>
-                <% end %>
-              </div>
+                </div>
+              <% end %>
             </div>
-          <% end %>
+          </div>
 
-          <div role="tabpanel" class="tab-pane <%= 'active' unless @discussion.has_submission? %>" id="content">
+          <div role="tabpanel" class="tab-pane" id="content">
```

Comment:
I'm a bit confused as to why were we adding a second `active` class for the `content` tab (the same happens with the `nav-link` further up), but this way is better as there should only be one active tab-pane at the time. I'm guessing the second one would override the first, but still, it was kinda messy...

Before merging, would you mind checking if all tabs fade consistently? Only the active tab-pane should have `show active`, but _all_ of them should have `fade` for the fade effect - or at least that's what the documentation and my quick testing suggest. I believe it should be on lines 76, 86 and 99 but please check it yourself.

See the examples here: https://getbootstrap.com/docs/5.0/components/navs-tabs/#javascript-behavior

Reply:
@felipecalvo I will see your frontend mumbo jumbo and raise you properly fading tabs!
![Peek 2021-09-09 14-14](https://user-images.githubusercontent.com/11720274/132732237-7fbeb4a4-4e6d-4ae8-82db-2784e824a2b1.gif)